src/java.compiler/share/classes/javax/lang/model/SourceVersion.java

```diff
@@ -223,7 +223,15 @@
      *
      * @since 16
      */
-    RELEASE_16;
+    RELEASE_16,
+
+    /**
+     * The version recognized by the Java Platform, Standard Edition
+     * 17.
+     *
+     * @since 17
+     */
+    RELEASE_17;
```

Comment:
Would it make sense to have a RELEASE_LATEST for the cases that are just updated to the latest release every six months?

Reply:
That kind of design was considered and rejected with the API was initially added. The use of enum constants in annotations must be an actual enum constant, not just a static final field pointing to a particular enum value. It would be possible to conceptually alias RELEASE_LATEST with whatever actual constant was the latest (16, then 17, then 18...), but that would cause issues with other uses of the API.